packages/apps/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -40,41 +45,36 @@ import { getExtensions } from './extensions';
 import meta, { MARKDOWN_PLUGIN } from './meta';
 import translations from './translations';
 import {
-  type ExtensionsProvider,
   type MarkdownPluginProvides,
   type MarkdownSettingsProps,
   MarkdownAction,
+  type MarkdownPluginState,
 } from './types';
 import { getFallbackTitle, isMarkdownProperties, markdownExtensionPlugins } from './util';
 
-export type MarkdownPluginState = {
-  // Codemirror extensions provided by other plugins.
-  extensions: NonNullable<ExtensionsProvider>[];
-};
-
 export const MarkdownPlugin = (): PluginDefinition<MarkdownPluginProvides> => {
   const settings = new LocalStorageStore<MarkdownSettingsProps>(MARKDOWN_PLUGIN, {
-    state: {},
     toolbar: true,
     experimental: false,
```

Comment:
minor thing for later API design but perhaps each module (i.e., plugin) has an experimental mode so that people can globally opt in to experimental (like an OS beta).